docs/framework/windows-workflow-foundation/deprecatedcontent-md.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Deprecate Content"
+description: DeprecateContent file, content to be included. 
+ms.date: "10/10/2020"
+---
+This material discusses types and namespaces that are obsolete. For more information, see [Deprecated Types in Windows Workflow Foundation 4.5](https://aka.ms/wfdeprecatedtypes).
```

Comment:
The aka.ms link redirects to https://docs.microsoft.com/en-us/dotnet/framework/windows-workflow-foundation/whats-new-in-wf-in-dotnet, which doesn't seem to be the correct link. We should just delete this file and remove the references to it, since it doesn't seem to add any value.

Reply:
will update